tests/antithesis/test-template/robustness/main.go

```diff
@@ -137,20 +138,49 @@ func testRobustness(ctx context.Context, lg *zap.Logger, r *report.TestReport, h
 }
 
 func runTraffic(ctx context.Context, lg *zap.Logger, hosts []string, baseTime time.Time, duration time.Duration) []report.ClientReport {
-	limiter := rate.NewLimiter(rate.Limit(profile.MaximalQPS), profile.BurstableQPS)
 	ids := identity.NewIDProvider()
-	storage := identity.NewLeaseIDStorage()
-	concurrencyLimiter := traffic.NewConcurrencyLimiter(profile.MaxNonUniqueRequestConcurrency)
-
 	r, err := traffic.CheckEmptyDatabaseAtStart(ctx, lg, hosts, ids, baseTime)
 	if err != nil {
 		lg.Fatal("Failed empty database at start check", zap.Error(err))
 	}
-
-	finish := closeAfter(ctx, duration)
 	reports := []report.ClientReport{r}
+	watchReport := []report.ClientReport{}
+	maxRevisionChan := make(chan int64, 1)
+	endpoints := make([][]string, len(hosts))
+	for _, host := range hosts {
+		endpoints = append(endpoints, []string{host})
+	}
```

Comment:
Let's simplify this function. This loop should not be needed if we only passed hosts to `CollectClusterWatchEvents`. This function doesn't need to accept `etcdEndpoints [][]string`, just `hosts []string` would be enough